packages/editor/src/components/media-placeholder/index.js

```diff
@@ -120,7 +100,7 @@ class MediaPlaceholder extends Component {
 					onFilesDrop={ this.onFilesUpload }
 					onHTMLDrop={ onHTMLDrop }
 				/>
-				{ ( onSelectUrl || onSelectURL ) && (
+				{ onSelectURL && (
```

Comment:
Right, based on this the function above should *error* if called without `onSelectURL` being specified.

Reply:
won't it silently skip this branch?